include/bluetooth/iso.h

```diff
@@ -34,6 +34,17 @@ extern "C" {
 				  BT_HCI_ISO_HDR_SIZE + \
 				  BT_HCI_ISO_DATA_HDR_SIZE)
 
+/** @def BT_ISO_SDU_BUF_SIZE
+ *
+ *  @brief Helper to calculate needed buffer size for ISO SDUs.
+ *         Useful for creating buffer pools.
+ *
+ *  @param mtu Required ISO SDU size
+ *
+ *  @return Needed buffer size to match the requested ISO SDU MTU.
+ */
+#define BT_ISO_SDU_BUF_SIZE(mtu) (BT_ISO_CHAN_SEND_RESERVE + (mtu))
```

Comment:
The definition of BT_ISO_CHAN_SEND_RESERVE could also be updated to say that it is the headrom needed for outgoing ISO SDUs. (similar to how l2cap definitions are documented).